src/gap.c

```diff
@@ -209,11 +209,11 @@ static Obj FuncSHELL(Obj self,
         outFile = "*stdout*";
     }
 
-    TypOutputFile output = { 0 };
```

Comment:
Being super-picky, I'm a tiny bit nervous we might not be setting some important value to 0 (or might not in the future).

Could we (hidden in a macro/function?) do a memset which sets everything except the 'line' buffer (which I see you moved to the end)

Reply:
I indeed moved the `line` buffer in an earlier version of this PR, for the very reason you mention, but then decided against it for now after I verified that we really init all fields as necessary.

Of course it is possible that we might add a new field in the future and then "forget" to initialize it, but I think there are bigger dangers in there?

Anyway, I'll think about it... What Id' also really like to *hide* that struct definition again, i.e. move it once more from the header to the C file...

Reply:
Maybe a better approach would be to insert a `memset(output, 0xFF, sizeof(output);` insider an `#ifdef` that enables it only in debug builds?